### PR TITLE
Add valid, invalid disabled text input examples

### PR DESCRIPTION
**Pull request by sukhrajghuman**

### CODE REVIEW

File: content/components/text-inputs/valid-invalid.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: component/code-demo
+iframe: examples/example-valid-invalid
+iframeFullwidth: true
+code:
+  - HTML: |
+      <!--
+        Light:  <input class="au-text-input au-text-input--valid">
+        Dark:   <input class="au-text-input au-text-input--valid au-text-input--dark">
+        Invalid:  <input class="au-text-input au-text-input--invalid">
```
Comment (alex-page) [inline content/components/text-inputs/valid-invalid.md]: Include `Invalid dark:` as well.

File: content/components/text-inputs/valid-invalid.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+      import AUtextInput from '@gov.au/text-inputs';
+
+      <label htmlFor="text-input" >label text</label>
+      <AUtextInput status="valid" />
+---
+
+## Valid and invalid inputs
+
+Add highlighting to text inputs and text areas to indicate whether user input is valid (validates according to the elements settings) or invalid (does not validate according to the elements settings).
```
Comment (alex-page) [inline content/components/text-inputs/valid-invalid.md]: I wouldn't refer to it as highlighting.

```suggestion
Add a border to text inputs and text areas to indicate whether user input is valid (validates according to the elements settings) or invalid (does not validate according to the elements settings).
```

File: content/components/text-inputs/disabled.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+layout: component/code-demo
+iframe: examples/example-disabled
+iframeFullwidth: true
+code:
+  - HTML: |
+      <!--
+        Light:  <input class="au-text-input" disabled>
+        Dark:   <input class="au-text-input au-text-input--dark" disabled>
+
```
Comment (alex-page) [inline content/components/text-inputs/disabled.md]: Unnecessary space here?
```suggestion
```

File: content/components/text-inputs/disabled.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      import AUtextInput from '@gov.au/text-inputs';
+
+      <label htmlFor="text-input" >label text</label>
+      <AUtextInput disabled />
+---
+
+## Disabled
+
+A disabled input element is unusable and not clickable. This can be used to prevent a user from interacting with the input element until another action is complete. Disabled input elements in a form will not be submitted.
```
Comment (alex-page) [inline content/components/text-inputs/disabled.md]: Some slight changes?

```suggestion
Disabled input elements are unusable and can not be clicked. This prevents a user from interacting with the input element until another action is complete. Disabled input elements in a form will not be submitted.
```

File: content/components/text-inputs/valid-invalid.md (hunk 1 above)
Comment (sukhrajghuman) [reply content/components/text-inputs/valid-invalid.md]: I think just invalid and invalid is good

File: content/components/text-inputs/disabled.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        Dark:   <AUtextInput dark>
+      */
+
+      import AUtextInput from '@gov.au/text-inputs';
+
+      <label htmlFor="text-input" >label text</label>
+      <AUtextInput disabled />
+---
+
+## Disabled
```
Comment (sukhrajghuman) [inline content/components/text-inputs/disabled.md]:
```suggestion
## Disabled text inputs
```

**Pull request merged**